Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
+	release-victoria-metrics-linux-amd64 \
+	release-victoria-metrics-darwin-amd64 \
 	release-victoria-metrics-arm \
+	release-victoria-metrics-linux-arm \
 	release-victoria-metrics-arm64 \
-	release-victoria-metrics-darwin-amd64 \
+	release-victoria-metrics-linux-arm64 \
 	release-victoria-metrics-darwin-arm64
 
 release-victoria-metrics-amd64:
 	OSARCH=amd64 $(MAKE) release-victoria-metrics-generic
 
+release-victoria-metrics-linux-amd64:
+	OSARCH=linux-amd64 $(MAKE) release-victoria-metrics-generic
+
+release-victoria-metrics-darwin-amd64:
+	OSARCH=darwin-amd64 $(MAKE) release-victoria-metrics-generic
+
 release-victoria-metrics-arm:
 	OSARCH=arm $(MAKE) release-victoria-metrics-generic
 
+release-victoria-metrics-linux-arm:
+	OSARCH=linux-arm $(MAKE) release-victoria-metrics-generic
+
 release-victoria-metrics-arm64:
 	OSARCH=arm64 $(MAKE) release-victoria-metrics-generic
 
-release-victoria-metrics-darwin-amd64:
-	OSARCH=darwin-amd64 $(MAKE) release-victoria-metrics-generic
+release-victoria-metrics-linux-arm64:
+	OSARCH=linux-arm64 $(MAKE) release-victoria-metrics-generic
 
 release-victoria-metrics-darwin-arm64:
 	OSARCH=darwin-arm64 $(MAKE) release-victoria-metrics-generic
 
-release-victoria-metrics-generic: victoria-metrics-$(OSARCH)-prod
+release-victoria-metrics-generic: \
+	victoria-metrics-$(OSARCH)-prod
```

Comment:
It is better from readability PoV leaving the dependent target on the line 134. The current code makes less clear that the `victoria-metrics-$(OSARCH)-prod` is a dependent target instead of the command, which must be executed by `release-victoria-metrics-generic`.